### PR TITLE
Update README

### LINKED ISSUE #16: Update repo readme
Update the repo readme to give basic information and links for users who visit this repo.

### PR DESCRIPTION
**Pull request by yymao**
This PR updates the repo README and fixes #16.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,7 @@
 # desc-data-portal
-Web app for LSSTDESC Public Data Access
+
+This repo hosts the code of the LSST DESC Data Portal web app and the asscoiated documentations and example notebooks.
+
+- Visit [lsstdesc-portal.nersc.gov](https://lsstdesc-portal.nersc.gov/) for the Data Portal and documentations.
```
Comment (heather999) [inline README.md:5]: maybe change "documentations" to "documention"?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,7 @@
 # desc-data-portal
-Web app for LSSTDESC Public Data Access
+
+This repo hosts the code of the LSST DESC Data Portal web app and the asscoiated documentations and example notebooks.
+
+- Visit [lsstdesc-portal.nersc.gov](https://lsstdesc-portal.nersc.gov/) for the Data Portal and documentations. 
+- Visit the [`notebooks` directory](notebooks) for example notebooks.
```
Comment (heather999) [inline README.md:6]: Is it worth explicitly stating these are Jupyter notebooks?

**Comment by yymao** [comment]
Thanks, Heather. I've addressed your comments and pushed the commit.

File: README.md (hunk 1 above)
Comment (heather999) [reply README.md]: sorry apparently I can't type today. I mean "documentation".

**Review by heather999 — APPROVED** [review]
LGTM

**Pull request merged**